docs/src/pages/docs/dynamic-forms.tsx

```diff
@@ -884,43 +589,26 @@ const ShowMeEverything: React.FC = () => {
     const sections: DynamicFormSection[] = [
         {
             key: "value-items-section",
-            title: "Simple single values only",
             items: valueItems,
             ...COMMON_SECTION,
         },
-        {
-            key: "multi-single-items-section",
-            title: "Choose one of many options",
-            items: multiSingleItems,
-            ...COMMON_SECTION,
-        },
-        {
-            key: "multi-many-items-section",
-            title: "Choose any of multiple options",
-            items: multiManyItems,
-            ...COMMON_SECTION,
-        },
     ];
 
-    const [renderForm, state, , setErrors] = useDynamicForm(sections);
+    const [renderForm, state, , setErrors, meta, touched] = useDynamicForm(sections);
+
+    console.log("touched: ");
+    console.log(touched);
+    console.log("META: ");
+    console.log(JSON.stringify(meta, null, 4));
+    console.log("--------");
```

Comment:
are these testing code? let's remove them if they are only meant for testing

Reply:
omg thanks I totally missed that

Reply:
looks like I ended up pushing the wrong file